server/routes/loginUser.js

```diff
@@ -0,0 +1,39 @@
+var express = require("express");
+var router = express.Router();
+var client = require("../database/connection");
+
+async function loginUser(client, user) {
+  result = await client
+    .db("memestagram")
+    .collection("users")
+    .findOne({ username: user.username, password: user.password })
+    .then(user => {
+      if (!user) {
+        console.log("404");
+        return false;
+      } else {
+        console.log("200");
+        return true;
+      }
+    });
+}
+
```

Comment:
Would you be able to add comments around these routing sections, so it is easier to read when looking back?

Reply:
Commented my code, except for login.js and signup.js as they are mostly standard react components

Reply:
> I notice that you are routing to different pages. However, would it be possible to display this information to the user (such as through the URL), so they know which page they are on?

Yeah I believe that functionality will come in afterwards, as currently the app is only at the point where we render the pages we want to test at root (just http://localhost:8000/). Once we get there, take a look at React's browser router: https://reacttraining.com/react-router/web/api